### ISSUE TITLE
Eval writing out JobClass emit function in result

### ISSUE THREAD

**Issue by zgohr** [node-js-libs/node.io#69]
```
$ echo "foo" | node.io -s eval "input.match(/bar/)"
```

Outputs this text to stdout

```
function (result) {
            worker.emit('output', result);
            instance.cancel_timeout();
            instance.isComplete = true;
            var emit = function () {

                job.instances--;

                //Reuse the instance later on
                job.instance_pool.push(instance);

                worker.emit('process');
            };

            if (job.options.wait) {
                setTimeout(emit, job.options.wait * 1000);
            } else {
                emit();
            }
        }
```

When I would have expected nothing to be returned.

**Comment by zgohr** [comment]
I have the fix, I'll make a pull request this afternoon

**Comment by chriso** [comment]
Fixed as of `v0.4.4`
